_posts/ericabrody/2016-02-03-Erica-Brody-Ch5-Exercises.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+author: ericabrody
+title: "Erica's Chapter 3 Exercises"
+---
+
+Exercise 1: Write a program which repeatedly reads numbers until the user enters “done”. Once “done” is entered, print out the total, count, and average of the numbers. If the user enters anything other than a number, detect their mistake using try and except and print an error message and skip to the next number.
+<iframe src="https://trinket.io/embed/python/e0f9320136" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Exercise 2: Write another program that prompts for a series of numbers as above and at the end prints out the maximum, minimum and average of the numbers.
+<iframe src="https://trinket.io/embed/python/fee86a045b" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflection:
+I know that these exercises aren't supposed to take long - as a beginning programmer, I spent about 90 minutes total figuring these out.
```

Comment:
Everyone's learning journey is different, so there's no 'expected time' to complete any exercise.  I'm glad you took the time to get your head around the problems.